### DISCUSSION TITLE
Custom type resolvers

### DISCUSSION THREAD

**Discussion by esseswann** [hasura/graphql-engine#3787] [General]
Is there any way I could customize type resolvers?
For example one might require resolve `timestampz` to Unix Timestamp or have it being able to parse formatting options like `timestamp(format: "moment-like format")`

**Comment by tirumaraiselvan** [comment]
You can use a scalar computed fields here? https://docs.hasura.io/1.0/graphql/manual/schema/computed-fields.html#computed-fields

**Comment by esseswann** [comment]
I don't think that this is same, you want to resolve specific fields in existing schema in some way, not add new fields

**Comment by tirumaraiselvan** [comment]
@esseswann Right. I was just proposing a workaround for now.
